File: C-C++/1010 - Calculo Simples.c
#include <stdio.h>

int main()
{
    int C, Q, i;
    double V, T = 0;

    for (i = 0; i < 2; i++)
    {
        scanf("%d", &C);
        scanf("%d", &Q);
        scanf("%lf", &V);
        T = (Q * V) + T;
    }

    printf("VALOR A PAGAR: R$ %.2lf\n", T);

    return 0;
}

File: JavaScript/1010_CalculoSimples.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});


const readLine = () => new Promise((resolve) => rl.question('', resolve));

async function processInput() {
    let total = 0;

    const line1 = await readLine();
    const [codigo1, num1, valor1] = line1.split(' ').map((x, i) => i > 0 ? parseFloat(x) : parseInt(x));
    total += num1 * valor1;

    const line2 = await readLine();
    const [codigo2, num2, valor2] = line2.split(' ').map((x, i) => i > 0 ? parseFloat(x) : parseInt(x));
    total += num2 * valor2;

    console.log(`VALOR A PAGAR: R$ ${total.toFixed(2).replace('.', ',')}`);

    rl.close();
}

processInput();
